roles/wordpress-setup/templates/wordpress-site.conf.j2

```diff
@@ -7,7 +7,7 @@ server {
   listen 80;
   {% endif %}
 
-  server_name  {{ item.value.site_hosts | join(' ') }};
+  server_name  {% for host in item.value.site_hosts %} {{ host }} {% if item.value.multisite.subdomains %} *.{{ host }} {% endif %} {% endfor %};
```

Comment:
:sunglasses: @fullyint @louim do you see any better way to do this? I came up with this under pressure and obviously it looks kinda dirty.